src/web/releases.rs

```diff
@@ -1060,4 +1060,12 @@ mod tests {
 
         assert_eq!(correct_json, serde_json::to_value(&release).unwrap());
     }
+
+    #[test]
+    fn release_feed() {
+        wrapper(|env| {
+            let web = env.frontend();
+            assert_success("/releases/feed", web)
```

Comment:
Can we also add tests for the content by adding fake releases? This can be done in a followup PR though.

Reply:
Yes, I'll add that in a follow-up.